static/app/components/featureBadge.tsx

```diff
@@ -1,6 +1,8 @@
-import * as React from 'react';
+import {Fragment, HTMLAttributes} from 'react';
```

Comment:
You can avoid importing `HTMLAttributes` if you just use `React.HTMLAttributes`, which is just in the global type namesapce